assets/css/v2/on_bus/time.scss

```diff
@@ -0,0 +1,44 @@
+.departure-time {
+  display: inline-block;
+  width: 100%;
+  margin-top: 25.5px;
+  font-size: 47px;
+  vertical-align: middle;
+}
+
+.departure-times-with-crowding,
+.departure-time-with-crowding {
+  height: 100%;
+}
+
+.departure-crowding {
+  display: none;
+}
+
+.departure-time__minutes {
+  display: inline-block;
+  width: 88px;
+  height: 100%;
+  font-weight: 800;
+  text-align: right;
+}
+
+.departure-time__minutes-label {
+  display: inline-block;
+  width: 42px;
+  height: 100%;
+  font-weight: 400;
+}
+
+.departure-time__text {
+  display: inline-block;
+  width: 100%;
+  font-weight: 800;
+  text-align: right;
+}
+
+.departure-time__timestamp {
+  width: 42px;
```

Comment:
Should this be like `__text` and be `width: 100%`, since like a `text` time it is the sole element in its container? Feels like they could share the exact same styles even.

```scss
.departure-time__text,
.departure-time__timestamp {
  width: 100%;
  height: 100%;
  font-weight: 400;
}
```

Reply:
True this does make sense to just be combined!